package.json

```diff
@@ -30,7 +30,8 @@
   "homepage": "https://github.com/macbre/phantomas-reporter-elasticsearch#readme",
   "dependencies": {
     "debug": "^2.2.0",
-    "elasticsearch": "^14.1.0"
+    "elasticsearch": "^14.1.0",
+    "public-ip": "^2.4.0"
```

Comment:
Is this dependency used here?

Reply:
no this dependency is unused here I just forget to remove it